.github/actions/auto-resolve-done-jira-issue/Initialize-IssueParameters.ps1

```diff
@@ -1,12 +1,10 @@
 param($Repository, $PullRequestNumber, $IsDone, $IsResolve)
 
-# We need to fetch the PR details from the API as opposed to just using the context, because the title may be changed
-# (by the user or the add-jira-issue-code-to-pull-request action) after the start of the run and that wouldn't be
-# present in it.
-$url = "https://api.github.com/repos/$Repository/pulls/$PullRequestNumber"
-$response = Invoke-WebRequest $url -Headers (Get-GitHubApiAuthorizationHeader) -Method Get
-$content = $response | ConvertFrom-Json
-$issueKey = Get-JiraIssueKeyFromPullRequestTitle $content.title
+# We need to fetch the PR details using the CLI (see https://cli.github.com/manual/gh_pr_view) as opposed to just using
+# the context, because the title may have changed (by the user or the add-jira-issue-code-to-pull-request action) after
+# the start of the run and that wouldn't be present in it.
+$title = gh pr view $PullRequestNumber --repo $Repository --json title --template '{{.title}}'
```

Comment:
Test this with the Jira project configured for ADHOC here: https://github.com/Lombiq/GitHub-CLI-Test

Reply:
works: https://github.com/Lombiq/GitHub-CLI-Test/pull/2